vassal-app/src/main/java/VASSAL/configure/TranslatingStringEnumConfigurer.java

```diff
@@ -113,6 +121,16 @@ public TranslatingStringEnumConfigurer(String key, String name, String[] validVa
     setValue(initialValue);
   }
 
+  public TranslatingStringEnumConfigurer(String key, String name, String[] validValues, String[] i18nKeys, char initialValue) {
+    this(key, name, validValues, i18nKeys, String.valueOf(initialValue));
+  }
+  public TranslatingStringEnumConfigurer(String[] validValues, String[] i18nKeys, String initialValue) {
+    this (null, null, validValues, i18nKeys, initialValue);
```

Comment:
Extra space.

Reply:
Tidied up this file and added missing Javadoc as well.